recipes/supervisor/post-link.sh

```diff
@@ -0,0 +1,6 @@
+if [[ $(id -u) == 0 ]]; then
+    mkdir -p /etc/supervisord/ && ln -s $PREFIX/etc/supervisord/ /etc/supervisord/
+    ln -s $PREFIX/etc/supervisord.conf /etc/supervisord.conf
```

Comment:
This is going to require sudo, right?  How can we make that gel with conda's user-level acess model?  Generally, if we're telling people to use sudo, they'll end up with files owned by root, which breaks later usability, then they devolve into using sudo for everything.  Imagine

    sudo ipython

Is there a way to not require sudo here?  I understand that it probably means not using supervisor at the system level - but it is still useful at the user level.  I use it for my raspberry pi photo frame at home this way.